pkgs/by-name/ha/hap-py/package.nix

```
@@ -0,0 +1,63 @@
+{ pkgs, stdenv, fetchFromGitHub, boost, zlib, lib, makeWrapper }:
```

```suggestion
{ stdenv, fetchFromGitHub, boost, zlib, lib, makeWrapper }:
```
Using pkgs here is dirty and breaks cross